From 47ff1eb9bb4f872c1d731b93d334ee5865bf3439 Mon Sep 17 00:00:00 2001
From: Denys Dmytriyenko <denys@ti.com>
Date: Sun, 6 Apr 2014 22:16:30 -0400
Subject: [PATCH] Fix tests Makefile usage of LDLIBS vs. LDFLAGS

Libraries must come after objects, as link order matters, especially
when using linker flags like -Wl,--as-needed.

Signed-off-by: Denys Dmytriyenko <denys@ti.com>

Upstream-Status: Pending
---
 tests/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/Makefile b/tests/Makefile
index cd202af..67c3c83 100644
--- a/tests/Makefile
+++ b/tests/Makefile
@@ -39,5 +39,5 @@ testprogs: $(hostprogs)
 clean:
 	rm -f *.o *~ $(hostprogs)
 
-${comp_progs}: LDFLAGS += -lssl -lcrypto
+${comp_progs}: LDLIBS += -lssl -lcrypto
 ${comp_progs}: %: %.o openssl_wrapper.o
-- 
1.9.1

